x/incentives/spec/02_state.md

```diff
@@ -0,0 +1,58 @@
+<!--
+order: 2
+-->
+
+# State
+
+## Incentives management
+
+All the incentives that is going to be provided are locked into `IncentivePool` until it's released to the users after a specific period of time.
+
+### Pot
+
+A `Pot` is a unit of incentive distributer. It describes the the users who can get reward, and the amount of coins to be distribute, the distribution start time, and the number of epochs it provide incentive from the `Pot`.
+
+```go
+type DistrCondition struct {
+  LockType   uint64 // type of lock, ByLockDuration | ByLockTime
```

Comment:
Could there be a note somewhere nearby for what the two lock types actually do?

Reply:
(Presumably by lock duration releases it linearly in relation to the time elapsed, whereas by lock time only releases it after the lock time is lifted)

Reply:
LockType is used as Enum and DistrCondition struct will be generated from protobuf.

Reply:
Might be worth investigating in the future if we can do some space savings optimizations here via oneof, but thats probably a minor concern for now.

Reply:
Maybe this should be called LockQueryType.  There's only one type of lock, just different ways of querying (by timestamp or duration)

Reply:
Ok let me update to LockQueryType